ReactWindows/ReactNative.Net46/ReactPage.cs

```diff
@@ -15,21 +15,35 @@ namespace ReactNative
     /// </summary>
     public abstract class ReactPage : Page, IAsyncDisposable
     {
-        private readonly IReactInstanceManager _reactInstanceManager;
-
-        private bool _isShiftKeyDown;
-        private bool _isControlKeyDown;
+        private readonly Lazy<IReactInstanceManager> _reactInstanceManager;
+        private readonly Lazy<ReactRootView> _rootView;
```

Comment:
I was not able to set the dispatcher in the App.xaml.cs because the DispatcherHelpers is internal and is not accessible... Since the dispatcher we want throughout is going to be the dispatcher on which the Page was instantiated we should be fine with the approach I took here... Ideally, we would probably want to find a way to trickle down dispatcher anywhere it could be needed... The DispatcherHelpers contract is the same for UWP and Net46 so the future implementation may involve an interface that is backed by two separate concrete implementations representing each platform...